changelog/3.7.0/3.7.0.md

```diff
@@ -0,0 +1,394 @@
+#### PDK
+
+- **PDK:** fix kong.request.get_forwarded_port to always return a number which was caused by an incorrectly
+stored string value in ngx.ctx.host_port.
+ [#12806](https://github.com/Kong/kong/issues/12806)
+ [KAG-4158](https://konghq.atlassian.net/browse/KAG-4158)
+
+- The value of `latencies.kong` in the log serializer payload no longer includes
+the response receive time, so it now has the same value as the
+`X-Kong-Proxy-Latency` response header. Response receive time is recorded in
+the new `latencies.receive` metric, so if desired, the old value can be
+calculated as `latencies.kong + latencies.receive`. **Note:** this also
+affects payloads from all logging plugins that use the log serializer:
+`file-log`, `tcp-log`, `udp-log`,`http-log`, `syslog`, and `loggly`.
+ [#12795](https://github.com/Kong/kong/issues/12795)
+ [KAG-3798](https://konghq.atlassian.net/browse/KAG-3798)
+
+- **Tracing**: enhanced robustness of trace ID parsing
+ [#12848](https://github.com/Kong/kong/issues/12848)
+ [KAG-4218](https://konghq.atlassian.net/browse/KAG-4218)
+#### Plugin
+
+- **AI-proxy-plugin**: Fix the bug that the route_type `/llm/v1/chat` does not include the analytics in the responses.
+ [#12781](https://github.com/Kong/kong/issues/12781)
+ [FTI-5769](https://konghq.atlassian.net/browse/FTI-5769)
+
+- **ACME**: Fixed an issue where the certificate was not successfully renewed during ACME renewal.
+ [#12773](https://github.com/Kong/kong/issues/12773)
+ [KAG-4008](https://konghq.atlassian.net/browse/KAG-4008)
+
+- **AWS-Lambda**: fix an issue that the latency attributed to AWS Lambda API requests will be counted as part of the latency in Kong
+ [#12835](https://github.com/Kong/kong/issues/12835)
+ [FTI-5261](https://konghq.atlassian.net/browse/FTI-5261)
+
+- **Jwt**: fix an issue where the plugin would fail when using invalid public keys for ES384 and ES512 algorithms.
+ [#12724](https://github.com/Kong/kong/issues/12724)
+
+
+- Add WWW-Authenticate headers to all 401 response in key auth plugin.
+ [#11794](https://github.com/Kong/kong/issues/11794)
+ [KAG-321](https://konghq.atlassian.net/browse/KAG-321)
+
+- **Opentelemetry**: fix otel sampling mode lua panic bug when http_response_header_for_traceid option enable
+ [#12544](https://github.com/Kong/kong/issues/12544)
+ [FTI-5742](https://konghq.atlassian.net/browse/FTI-5742)
+#### Admin API
+
+- **Admin API**: fixed an issue where calling the endpoint `POST /schemas/vaults/validate` was conflicting with the endpoint `/schemas/vaults/:name` which only has GET implemented, hence resulting in a 405.
+ [#12607](https://github.com/Kong/kong/issues/12607)
+ [KAG-3699](https://konghq.atlassian.net/browse/KAG-3699)
+#### Default
+
+- Fix a bug where the ulimit setting (open files) is low Kong will fail to start as the lua-resty-timer-ng exhausts the available worker_connections. Decrease the concurrency range of the lua-resty-timer-ng library from [512, 2048] to [256, 1024] to fix this bug.
```

Comment:
```suggestion
- Fixed a bug where, if the the ulimit setting (open files) was low, Kong would fail to start as the `lua-resty-timer-ng` exhausted the available `worker_connections`. Decreased the concurrency range of the `lua-resty-timer-ng` library from `[512, 2048]` to `[256, 1024]` to fix this bug.
```